include/vara/Feature/Constraint.h

```diff
@@ -113,6 +115,12 @@ class OrConstraint : public BinaryConstraint, public BooleanConstraint {
       : BinaryConstraint(ConstraintKind::CK_OR, std::move(LeftOperand),
                          std::move(RightOperand)) {}
 
+  std::unique_ptr<Constraint> clone() override {
+    return std::make_unique<OrConstraint>(
+        std::move(this->getLeftOperand()->clone()),
+        std::move(this->getRightOperand()->clone()));
```

Comment:
```suggestion
        this->getLeftOperand()->clone(),
        this->getRightOperand()->clone());
```
The returned value is already an rvalue